next/assets/ui-icons/index.ts

```diff
@@ -52,3 +52,6 @@ export { default as AlertIcon } from './vykricnik-trojuholnik.svg'
 export { default as RemoveIcon } from './vymazat-plny-kruh.svg'
 export { default as LockIcon } from './zamok.svg'
 export { default as CrossInCircleIcon } from './zrusit-kruh.svg'
+export { default as AddressIcon } from './adresa.svg'
+export { default as WebIcon } from './web.svg'
+export { default as EmailIcon } from './e-mail.svg'
```

Comment:
UI icons are exported from this figma. Please do not mix them with other icons, create separate folder or put them into `images`.

https://www.figma.com/file/lmi5bRuoosgzeOhnmg4Zkk/DS-ESBS%3A-Custom-icons?type=design&node-id=0-1&t=0F4ZUkg8ZN7g97Mr-0